zulipterminal/ui.py

```diff
@@ -79,6 +79,11 @@ def footer_view(self) -> Any:
         text_header = self.get_random_help()
         return urwid.AttrWrap(urwid.Text(text_header), 'footer')
 
+    def show_updated_right_column(self, model: Any) -> Any:
+        self.users = model.users
+        self.right_panel = self.right_column_view()
+        super(View, self).__init__(self.build_window_frame())
```

Comment:
Is this necessary? What are you trying to do?

Reply:
I am trying to call only the `build_window_frame()` method here. As both the left and middle windows are already built. However only the right panel - which is the user's list has to be refreshed (built again), but now with the new user's list.

Reply:
I was trying to understand why you were using `super`, really.

That said, can you just use `update_screen()` instead of splitting the function and rebuilding it each time? That is, do you really need the commit that splits the method and makes `build_window_frame`?

Reply:
I did this in the current iteration. Did away with the first commit, using `update_screen()` to reflect the changes in the Right User column. I also replaced the `super` call with a normal one. However the changes don't seem to work. I am a bit puzzled here. We are building the UI every 60 seconds here?